components/version_info/version_info.cc

```diff
@@ -8,7 +8,9 @@
 #include "base/no_destructor.h"
 #include "base/version.h"
 #include "build/build_config.h"
+#if !STARBOARD
 #include "components/version_info/version_info_values.h"
+#endif
```

Comment:
consider doing this:
```
#else
#include "components/version_info/starboard_version_info_values.h"
```

And provide ( initially hardcoded ) definitions for PRODUCT_NAME, PRODUCT_VERSION, IS_OFFICIAL_BUILD etc in there. This will provide a cleaner intercept, and allows us to plug in the proper generation mechanism from build tooling side later with fewer changes.

Reply:
Done.